guides/common/assembly_preparing-environment-for-installation.adoc

```diff
@@ -1,25 +1,9 @@
 ifdef::context[:parent-context: {context}]
 
-[id="Preparing_your_Environment_for_Installation_{context}"]
-= Preparing your environment for installation
+[id="Preparing_Environment_{context}"]
+= Preparing Environment
```

Comment:
Sentence case, so lowercase `e`.

Reply:
Also, I can see that this assembly includes the module on IPv4 requirements. Can you move the prerequisites for IPv6 requirements here as well? That's `guides/common/assembly_preparing-environment-for-installation-in-ipv6-network.adoc`.